### PR TITLE
Update Set-VMComPort.md 

### PR DESCRIPTION
**Pull request by namkedia**
It is not clear if one has to create a named pipe or how this connection to the COM port can then be put to use.

### CODE REVIEW

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing and style checks needed for this PR. Thanks.

**Comment by e0i** [comment]
@get-itips @tseward

Peer review needed for this PR. Thanks.

**Comment by tseward** [comment]
This needs to be in [!NOTE] format.

**Comment by get-itips** [comment]
Agree with @tseward

File: docset/windows/hyper-v/Set-VMComPort.md [hunk 1]
```diff
@@ -56,6 +56,8 @@ PS C:\> Set-VMComPort TestVM 2 \\.\pipe\TestPipe
 
 Configure the second COM port on virtual machine TestVM to connect to named pipe TestPipe on the local computer.
 
+Note: This does not create the named pipe on the local computer. You will have to write your own application to create a named pipe.
```
Comment (JohanFreelancer9) [inline docset/windows/hyper-v/Set-VMComPort.md]:
```suggestion
> [!NOTE]
> This does not create the named pipe on the local computer. You will have to write your own application to create a named pipe. 
```

**Comment by e0i** [comment]
@namkedia Would you be able to follow-up with copy-editing suggestions? We would like to proceed with merge. Thanks.

**Comment by e0i** [comment]
Replacing with follow-up PR #1630

**Pull request closed without merging**